blocks_vertical/control.js

```diff
@@ -61,8 +61,7 @@ Blockly.Blocks['control_repeat'] = {
       "args0": [
         {
           "type": "input_value",
-          "name": "TIMES",
```

Comment:
In 2.0 it does, I was just making it match for now. We're having lots of questions around here about what should snap into where, but I figured it would make a good starting point to have us match the current behavior :)